HPCC-22772 Cannot make install 7.6.x debug

miguelvazq opened the pull request:
Rolling back the addition of React components. When trying to remove the
node_modules/preact directory (rimraf) anyone running as sudo was
causing conflicts between preact vs. react typings. We will create a react
component a bundle to eliminate any conlusion with other libraries that
may cause conflicts in the future.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexisrisk.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

HPCCSmoketest commented:
Process of PR-12956, label: HPCC-22772 starts now.
The reason of this test is: Did not build yet.
Commit ID: 29e85606fede6b6b87a07614e5c0e7fb6d79f1c6
Estimated completion time is ~0.1 hour(s)

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
OS: centos 7.6.1810 (Linux 3.10.0-327.28.3.el7.x86_64)
 GCC:gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
Sha: 29e85606fede6b6b87a07614e5c0e7fb6d79f1c6
Build: success
Build: success
Install HPCC Platform
HPCC Start: OK
HPCC Stop: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   49 sec (00:00:49) |  249 sec (00:04:09) |    0 sec (00:00:00) |   73 sec (00:01:13) |   24 sec (00:00:24) |    0 sec (00:00:00) |   15 sec (00:00:15) |  410 sec (00:06:50) |

miguelvazq commented:
@GordonSmith Please review. @kunalaswani did it build using sudo make?

kunalaswani commented:
Cherry-picked, built and tested using sudo make, and installed fine. Working correctly.

GordonSmith reviewed (approved):
Looks good.

richardkchapman commented:
@miguelvazq Can you rebase/retarget to 7.6.0 please

Commit:
HPCC-22772 Cannot make install 7.6.x debug

Rolling back the addition of React components. When trying to remove the
node_modules/preact directory (rimraf) anyone running as sudo was
causing conflicts between preact vs. react typings. We will create a
react component a bundle to eliminate any conlusion with other libraries that
may cause conflicts in the future.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexisrisk.com>

miguelvazq commented:
@richardkchapman Should be good now.

HPCCSmoketest commented:
Process of PR-12956, label: HPCC-22772 starts now.
The reason of this test is: New commit.
Commit ID: 9736ad876b0f70fe7e2b064e3e6df8a27cbe0349
Estimated completion time is ~0.1 hour(s)

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
OS: centos 7.6.1810 (Linux 3.10.0-327.28.3.el7.x86_64)
 GCC:gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
Sha: 9736ad876b0f70fe7e2b064e3e6df8a27cbe0349
Build: success
Build: success
Install HPCC Platform
HPCC Start: OK
HPCC Stop: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   52 sec (00:00:52) |  252 sec (00:04:12) |    0 sec (00:00:00) |   73 sec (00:01:13) |   18 sec (00:00:18) |    0 sec (00:00:00) |   22 sec (00:00:22) |  417 sec (00:06:57) |
